.changeset/silly-suns-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/util': minor
```

Comment:
Oh changeset-checker finally caught this for once. We'll also need a `'firebase': minor`.